.buildkite/scripts/steps/scalability/download_artifacts.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+mkdir -p "$SCALABILITY_ARTIFACTS_LOCATION"
+
+gsutil cp "$GCS_BUCKET/LATEST" "$SCALABILITY_ARTIFACTS_LOCATION/"
+HASH=`cat $SCALABILITY_ARTIFACTS_LOCATION/LATEST`
+gsutil cp -r "$GCS_BUCKET/$HASH" "$SCALABILITY_ARTIFACTS_LOCATION/"
+
+cd "$SCALABILITY_ARTIFACTS_LOCATION/$HASH"
+echo "Upload scalability traces as build artifacts"
+buildkite-agent artifact upload "scalability_traces.tar.gz"
```

Comment:
This script does not only download artifacts but also uploads the traces as build artifact. We should therefore either rename the script or reduce its scope to really just download artifacts and do the build artifact upload separately. In the latter case I'd suggest to put all build artifacts in a common directory and have more generic code upload whatever is in that directory.

Reply:
I moved it to the end of script, `upload_test_results` function